includes/Core/Dismissals/REST_Dismissals_Controller.php

```diff
@@ -95,6 +96,39 @@ protected function get_rest_routes() {
 					'permission_callback' => $can_dismiss_item,
 				)
 			),
+			new REST_Route(
+				'core/user/data/dismissed-items',
+				array(
+					'methods'             => WP_REST_Server::DELETABLE,
+					'callback'            => function ( WP_REST_Request $request ) {
+						$slugs = $request['data']['slugs'];
+
+						foreach ( $slugs as $slug ) {
+							$this->dismissed_items->remove( $slug );
+						}
+
+						return new WP_REST_Response( $this->dismissed_items->get_dismissed_items() );
+					},
+					'permission_callback' => $can_dismiss_item,
+					'args'                => array(
```

Comment:
I've diverged from the IB to use JSON schema validation for the REST parameters, rather than manual validation in the handler.